packages/sandbox/test/twoslashSupport.test.ts

```diff
@@ -30,7 +30,7 @@ fn(42)
 
   it("ignores unknown vars", () => {
     const sandbox = sandboxMock(`
-// @noImplicny: false
+// @noImplicitAny: false
```

Comment:
```suggestion
// @noImplicny: false
```

This looks to be intentional for this test; see failing tests.

Reply:
The comment below indicated it was expecting to the correct suppression name

Reply:
Yes, but the `compilerOptions` snapshot below it show that `noImplicny` is dropping the property.